samples/client/petstore/go/go-petstore-withXml/model_outer_enum.go

```diff
@@ -13,7 +13,7 @@ type OuterEnum string
 
 // List of OuterEnum
 const (
-	PLACED OuterEnum = "placed"
-	APPROVED OuterEnum = "approved"
-	DELIVERED OuterEnum = "delivered"
+	OuterEnum_PLACED OuterEnum = "placed"
```

Comment:
Would it be better if it's all upper case, e.g. OUTERENUM_PLACED?

Ref: https://stackoverflow.com/a/22688926/677735

We also need an option so that users can stick with the current behavior instead (e.g. PLACED instead of OUTERENUM_PLACED)

Reply:
Well, the Go guides are pretty adamant about using mixed case, but I elected not to change the part that was already there that didn't. But I can uppercase the typename for consistency.

Unfortunately I doubt I'll dig into the code enough to learn how to add an option flag.

Reply:
I think I can help on that.